scripts-python/create_test

```diff
@@ -123,11 +123,10 @@ formatter_class=argparse.ArgumentDefaultsHelpFormatter
                         help="Number of tasks create_test should perform simultaneously. Default "
                         "will be min(num_cores, num_tests).")
 
-    parser.add_argument("--xml_machine",help="Use this machine key in the lookup in testlist.xml")
-    parser.add_argument("--xml_compiler",help="Use this compiler key in the lookup in testlist.xml")
-    parser.add_argument("--xml_category",help="Use this category key in the lookup in testlist.xml")
-    parser.add_argument("--xml_testlist",help="Use this testlist to lookup tests")
-
+    parser.add_argument("--xml-machine",  help="Use this machine key in the lookup in testlist.xml")
```

Comment:
Backwards compatibility with the Perl create_test is already lost due to the double-dash requirement, so we may as well be consistent with the other options.